### PR TITLE
Try another plugin

### PR DESCRIPTION
**Pull request by wawrzek**
This time automate tag creation

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -37,9 +37,20 @@ jobs:
           echo '### Enclave Image Measurement' >> $GITHUB_STEP_SUMMARY
           cat target/release/enclave_measurement.txt >> $GITHUB_STEP_SUMMARY
 
+      - name: Bump version and push tag
```
Comment (shamsasari) [inline .github/workflows/release.yml:40]: Can we not bump the version

Comment (wawrzek) [reply .github/workflows/release.yml:40]: I think it's going to push a new tag as a version. Let's try it, I don't know what's going to happen. 

BTW. What's wrong with bumping version from 0.0.45 to 0.0.46 etc?

Comment (shamsasari) [reply .github/workflows/release.yml:40]: >BTW. What's wrong with bumping version from 0.0.45 to 0.0.46 etc?

On every single commit to `main`? Seems pointless. Why not just use the git commit. Also, every time a PR is merged, there will be two commits to main.

Comment (wawrzek) [reply .github/workflows/release.yml:40]: 1) with recent change, I don't think it bumps the version
2) I don't care about numbers, I care about experience, and I don't have time to spend on a perfect solution. If we want every change on main to be reflected on a test system, we need this solution. At least for now.
3) I'm happy to create fewer releases, by bind automation of their creation to manual tag creation.

I'm going to merge this PR and let see what's going to happen (if anything). We can refine the process down the line.

**Pull request merged**